www/docs/mp/index.php

```diff
@@ -606,6 +617,37 @@
 
                 break;
 
+            case 'divisions':
+                $policyID = get_http_var('policy');
+                $answered_q = get_http_var('answered');
+                $policiesList = new MySociety\TheyWorkForYou\Policies;
+                $positions = new MySociety\TheyWorkForYou\PolicyPositions( $policiesList, $MEMBER );
+
+                $data['user_code'] = bin2hex(urandom_bytes(16));
+                $data['auth_signature'] = auth_sign_with_shared_secret($data['user_code'], OPTION_SURVEY_SECRET);
+
+                $data['page_url'] = "http://" . DOMAIN . $_SERVER['REQUEST_URI'] . ( $policyID ? '&' : '?' ) . 'answered=1';
+                $data['answered_q'] = $answered_q;
+
+                $data['positions'] = $positions->positionsById;
```

Comment:
Sorry, I probably wasn't clear. I'm not suggesting getting rid of that page, just that the code seems to have two ways of doing the same thing - restricting Policies before passing to PolicyPositions, or not restricting and using this new positionsById.